File: ktpm-front-angular/src/model/FoodOrder.ts
export class FoodOrder {
  id: number;
  foodId: number;
  quantity: number;
  name: string;
  price: number;
  billId: number;

  constructor(data: any) {
    this.id = data.id ?? 0; // Sử dụng 0 nếu id không được cung cấp
    this.foodId = data.foodId;
    this.quantity = data.quantity;
    this.name = data.name;
    this.price = data.price;
    this.billId = data.billId;
  }
}

File: KTPM/src/main/java/com/example/ktpm/model/FoodOrder.java
package com.example.ktpm.model;

import jakarta.persistence.*;

@Entity
public class FoodOrder {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @Column(name = "foodId")
    private Long foodId;
    @Column(name = "quantity")
    private Long quantity;
    @Column(name = "name")
    private String name;
    @Column(name = "price")
    private Long price;
    @Column(name = "billId")
    private Long billId;
    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Long getFoodId() {
        return foodId;
    }

    public void setFoodId(Long foodId) {
        this.foodId = foodId;
    }

    public Long getQuantity() {
        return quantity;
    }

    public void setQuantity(Long quantity) {
        this.quantity = quantity;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Long getPrice() {
        return price;
    }

    public void setPrice(Long price) {
        this.price = price;
    }

    public Long getBillId() {
        return billId;
    }

    public void setBillId(Long billId) {
        this.billId = billId;
    }
}
